specification/ilm/_types/Phase.ts

```diff
@@ -17,14 +17,17 @@
  * under the License.
  */
 
-import { Time } from '@_types/Time'
+import { TimeSpan } from '@_types/Time'
 import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
 import { RolloverConditions } from '@indices/rollover/types'
-import { integer } from '@_types/Numeric'
+import { integer, long } from '@_types/Numeric'
 
 export class Phase {
   actions?: Actions
-  min_age?: Time
+  /**
+   * @es_quirk output as a millis number in XPack usage stats, which cannot roundtrip with a TimeSpan as it requires a unit.
```

Comment:
Is this an actual quirk with Elasticsearch's API or is this due to the way we're choosing to implement this type?

Reply:
It's an actual quirk. ES will not accept as input of `PUT /_ilm/policy/{name}` the ILM policy output by `GET _xpack/usage`. Not sure many users will try to do this instead of using the output of `GET _ilm/policy` that is correctly formatted, but if they do this will not work.  I opened https://github.com/elastic/elasticsearch/issues/87327 for this.